### PR TITLE
dibs: implement exercise two-bucket

### PR DESCRIPTION
**Pull request by cptjackson**
Time for another one.

-----
Solves #734

### CODE REVIEW

**Comment by cptjackson** [comment]
Not sure what has gone wrong here. Am I trying to merge the wrong thing? Still a bit inexperienced with git.

**Comment by snahor** [comment]
Probably you created your `two-bucket` from an outdated branch (`scale-generator`?), you should have used `master`. Well, you have a lot options such as `git merge`, `git rebase`, `git replace`, etc. Since you mentioned that you are inexperienced, the easiest way that comes to my head would be:
```
# in your two-bucket branch,  you will use that output later
git rev-parse HEAD

# go to your master branch
git checkout master

# update it

# rename your two-bucket branch to temp (or whatever you want)
git branch -m two-bucket temp

# create a new two-bucket branch from master
git checkout -b two-bucket

# add your commit to this new branch
git cherry-pick <hash from the first step>

# update your branch on github
git push origin +two-bucket
```

**Comment by behrtam** [comment]
@cptjackson would you mind following @snahor tips so this PR gets cleaned up and someone can review it?

**Comment by cptjackson** [comment]
Yes, apologies – this slipped down my priority list. Will aim to fix this weekend.

**Comment by cptjackson** [comment]
Hi – tried this out but got the following error and hints:

`could not apply 3ac856f... Add exercise two-bucket
hint: after resolving the conflicts, mark the corrected paths
hint: with 'git add <paths>' or 'git rm <paths>'
hint: and commit the result with 'git commit'`

When I switched to master it said that my branch was up to date, so there was no update step

**Comment by snahor** [comment]
What have you tried? merge/rebase? In order to update your master
branch you have to fetch from exercism/python.
Hans Roman

File: config.json [hunk 1]
```diff
@@ -489,6 +489,22 @@
       ]
     },
     {
+      "slug": "two-bucket",
+      "difficulty": 5,
```
Comment (m-a-ge) [inline config.json]: `uuid` is required

File: config.json [hunk 2]
```diff
@@ -489,6 +489,22 @@
...
+      "topics": [
+        "strings",
+        "pattern matching"
+      ]
+    },
+    {
+      "slug": "scale-generator",
+      "difficulty": 3,
```
Comment (m-a-ge) [inline config.json]: `uuid` is required

**Comment by m-a-ge** [comment]
This PR seems like a lot of work, it would be awesome to have it in `master`

@cptjackson, do you implemented both `scale_generator` and `two-bucket` exercises or I'm missing something?

**Comment by snahor** [comment]
@m-a-ge his `scale-generator` is already merged. The problem is that he used it as the root for his `two-bucket` branch.

@cptjackson could you give me write access to your fork? I've fixed your conflicts.

**Comment by m-a-ge** [comment]
@cptjackson, @snahor, guys, are you still interested in finishing this?

**Comment by m-a-ge** [comment]
two-bucket was added in #760

**Comment by m-a-ge** [comment]
Will leave this open since therese almost complete implementation of `scale-generator` - 
```text
exercises/scale-generator/example.py
exercises/scale-generator/scale_generator.py
exercises/scale-generator/scale_generator_test.py
```

**Comment by N-Parsons** [comment]
Closing because `scale-generator` was implemented and merged in #474, and `two-bucket` has since been implemented and merged in #760.

**Comment by m-a-ge** [comment]
Thanks, @N-Parsons

**Pull request closed without merging**